docs/application/native/guides/text-input/autofill.md

```diff
@@ -199,7 +183,7 @@ To get the presence of Autofill data and Autofill authentication information:
     autofill_view_info_destroy(vi_h);
     ```
 
-## Data entered and Stored in Autofill Service
+## Data Entered and Stored in Autofill Service
```

Comment:
Line 171: Request with the `autofill_auth_info_request()` function, to get the authentication information:

Reply:
Fixed.

Reply:
Fixed.